routes/originGitHub.js

```diff
@@ -55,7 +55,7 @@ router.get(
 )
 
 router.get(
-  '/:login/:repo?',
+  '/:login{/:repo}',
```

Comment:
how to handle the optional repo? /origins/github/aws/java and /origins/github/aws should both work here

Reply:
Thanks for the fix and adding the related tests!